### PR TITLE
Changes

### PR DESCRIPTION
**Pull request by margoroma**
move folders

### CODE REVIEW

**Comment by margoroma** [comment]
Самостоятельная, задание 4.

**Comment by margoroma** [comment]
added Test_classwork

File: Ramanenka_Tasks/Test1/RM_ClassWork_zadanie4.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
+stroka = 'five thirteen two eleven seventeen two one thirteen ten four eight five nineteen'
+new_list = stroka.split(' ')
+print(f'Список из строки - {new_list}')
+chisla = [5, 13, 2, 11, 17, 2, 1, 13, 10, 4, 8, 5, 19]
```
Comment (MikitaTsiarentsyeu) [inline Ramanenka_Tasks/Test1/RM_ClassWork_zadanie4.py:4]: and what if those numbers in the string were presented in a different order?

File: Ramanenka_Tasks/Test1/RM_ClassWork_zadanie4.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+for i in range(len(chisla)):                    # В числа
+    new_list[i] = chisla[i]
+print(f'Преобразованный в числа список - {new_list}')  
+
+
+new_list_uniq = []                              # Убрать повторы
+for i in new_list:
+    if i not in new_list_uniq:
+        new_list_uniq.append(i)
```
Comment (MikitaTsiarentsyeu) [inline Ramanenka_Tasks/Test1/RM_ClassWork_zadanie4.py:15]: totally not optimal solution

File: Ramanenka_Tasks/Test1/RM_ClassWork_zadanie4.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+print(f'Список без повторов - {new_list_uniq}')     
+          
+
+for i in range(len(new_list_uniq)):             # Сортировка по возрастанию
+    mini = i
+    for j in range(i+1, len(new_list_uniq)):
+        if new_list_uniq[j] < new_list_uniq[mini]:
+            mini = j
+    new_list_uniq[mini], new_list_uniq[i] = new_list_uniq[i], new_list_uniq[mini]
```
Comment (MikitaTsiarentsyeu) [inline Ramanenka_Tasks/Test1/RM_ClassWork_zadanie4.py:25]: you could use standard python sorting here actually

File: Ramanenka_Tasks/Test1/RM_ClassWork_zadanie4.py [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+print(f'Возрастающий список - {new_list_uniq}')
+
+
+print(f'Математические операции с элементами списка (построчно - сумма, произведение и т.д.):')
+for i in range(3):                              # Мат. операции
```
Comment (MikitaTsiarentsyeu) [inline Ramanenka_Tasks/Test1/RM_ClassWork_zadanie4.py:31]: wtf is range(3)? rework it to be range of your list in general case

**Review by MikitaTsiarentsyeu — CHANGES_REQUESTED** [review]

**Pull request closed without merging**